crypto/asn1/a_int.c

```diff
@@ -265,7 +265,7 @@ static int asn1_get_int64(int64_t *pr, const unsigned char *b, size_t blen,
             *pr = -(int64_t)r;
         } else if (r == ABS_INT64_MIN) {
             /* This never happens if INT64_MAX == ABS_INT64_MIN, e.g.
```

Comment:
Fix the code style errors for the comments..
Correct format is..
/*
 * Line 1
 * Line 2
 */